reference/supported-rendering-resources.md

```diff
@@ -50,7 +52,18 @@ The `webrtc` platform is used by Anbox to provide graphical output. It supports
 
 ## Supported APIs
 
-For NVIDIA GPUs Anbox Cloud uses [OpenGL ES 3.2](https://www.khronos.org/opengles/) and [EGL 1.5](https://www.khronos.org/egl/) graphical interfaces. For Intel and AMD GPUs, Anbox Cloud uses [Vulkan 1.3](https://vulkan.org/).
+| API        | Version | Supported GPUs     |
+|------------|---------|--------------------|
+| EGL        | 1.5     | AMD, Intel, NVIDIA |
+| OpenGL ES  | 3.2     | AMD, Intel, NVIDIA |
+| Vulkan     | 1.3     | AMD, Intel, NVIDIA |
```

Comment:
I thought Vulkan 1.3 will only be supported starting with 1.22?

Reply:
On the Android client side, yes. But we support Vulkan 1.3 drivers on the bottom of that.